### PR TITLE
Edit and add JUnit test for SeeAllExpensesCommand

### PR DESCRIPTION
**Pull request by tsoudibi**
Add comments to the code.
As mentioned in #49, change the message for SeeAllExpensesCommand when no expenses are logged.

### CODE REVIEW

**Comment by tsoudibi** [comment]
Add JUnit tests for SeeAllExpensesCommand

File: src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.financial.Expense;
+import seedu.duke.financial.FinancialEntry;
+import seedu.duke.financial.FinancialList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```
Comment (limkongkiat) [inline src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java:12]: Any reason for having so many line spaces here?

File: src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+
+public class SeeAllExpensesCommandTest {
```
Comment (limkongkiat) [inline src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java:16]: Perhaps adding Javadoc comments for the class would be good?

File: src/main/java/seedu/duke/command/SeeAllExpensesCommand.java
Comment (limkongkiat) [inline src/main/java/seedu/duke/command/SeeAllExpensesCommand.java:1]: LGTM

**Review by limkongkiat — COMMENTED** [review]
LGTM overall, just a few nits in the Test class

File: src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java (hunk 2 above)
Comment (tsoudibi) [reply src/test/java/seedu/duke/command/SeeAllExpensesCommandTest.java:16]: Yeah of course. Working on it.

**Pull request merged**